_security-plugin/configuration/multi-auth.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: default
+title: Multi-option sign-in
+parent: Configuration
+nav_order: 3
+---
+
+# Multi-option sign-in for Dashboards
+
+You can configure the sign-in window for OpenSearch Dashboards to provide either a single option for authenticating users at login or multiple options. Currently, Dashboards supports basic authentication, OpenID Connect, and SAML as choices for multi-option authentication.
+
+## General steps for configuring multi-option sign-in
+
+1. Decide which types of authentication to make available at sign-in.
+1. Configure each authentication type, including an authentication domain for the identification provider (IdP) and the essential settings that give each type sign-in access to OpenSearch Dashboards (see steps for each authentication type in the Configuration section of the security documentation).
+1. Add, enable, and configure multi-option authentication sign-in settings in the `opensearch_dashboards.yml` file.
+
+## Enabling multi-option sign-in
+
+By default, Dashboards provides a single sign-in environment for basic authentication. To enable multiple options for authentication at sign-in, begin by adding `opensearch_security.auth.multiple_auth_enabled` to the `opensearch_dashboards.yml` file and setting it to `true`.
```

Comment:
`single sign-in environment` can cause some confusion with 'single sign-on' using external IDP. We may consider to line up the wording with `multiple options`.